starter-code/index.html

```diff
@@ -1,101 +1,106 @@
-    <link rel="stylesheet" href="vendor/styles/normalize.css">
-    <link rel="stylesheet" href="vendor/icons.css">
-    <link rel="stylesheet" href="styles/base.css">
-    <link rel="stylesheet" href="styles/layout.css">
-    <link rel="stylesheet" href="styles/modules.css">
-    <!-- Code Fellows favicon -->
-    <link rel="icon" type="image/x-icon" href="https://i.imgur.com/7v5ASc8.png">
-    <!--
+<head>
+  <meta charset="UTF-8">
+  <title>Blog</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="vendor/styles/normalize.css">
+  <link rel="stylesheet" href="vendor/icons.css">
+  <link rel="stylesheet" href="styles/base.css">
+  <link rel="stylesheet" href="styles/layout.css">
+  <link rel="stylesheet" href="styles/modules.css">
+  <!-- Code Fellows favicon -->
+  <link rel="icon" type="image/x-icon" href="https://i.imgur.com/7v5ASc8.png">
+  <!--
   TODO: Complete this script tag below to hold the Handlebars template.
-    1. Be sure to give it a meaningful id attribute, and don't forget to
-      specify the type!
-    2. Insert correct expressions properties based on our Model.
-    -->
-    <script id="" type="">
-      <article class="clearfix" data-author="" data-category="">
-        <header>
-          <h1></h1>
-          <div class="byline">
-            By <address><a href=""></a></address>
-            {{publishStatus}}
-          </div>
-        </header>
-        <section class="article-body"></section>
-        <a href="#" class="read-on">Read on &rarr;</a>
-      </article>
-    </script>
-    <!--
-        STRETCH! Look at all of the duplicated code in our
-        populateFilters method! Looks like a good opportunity
-        to use a template. Make two small templates
-        (one for author, one for category) each with an
-        option element, and render the list once you have data
-        to populate it with.
-       -->
-  </head>
-  <body>
-    <header class="site-header clearfix">
-      <section class="heading-group-h1">
-        <h1>
-          Kilovolt Blog!
-        </h1>
-        <h2>The shocking power of static pages</h2>
-      </section>
-      <nav class="main-nav">
-        <!-- Hamburger Icon -->
-        <div class="icon-menu"></div>
+  Be sure to give it a meaningful id attribute, and don't forget to
+  specify the type!
+  Insert correct expressions properties based on our Model.
+-->
+<script id="article-template" type="text/x-handlebars-template">
+  <article class="clearfix" data-author="{{author}}" data-category="{{category}}">
+    <header>
+      <h1>{{title}}</h1>
+      <div class="byline">
+        By <address><a href="{{authorUrl}}"></a></address>
+        {{author}}
+        {{category}}
+        {{authorurl}}
+        {{publishedOn}}
+        {{{body}}}
+      </div>
+    </header>
+    <section class="article-body"></section>
+    <a href="#" class="read-on">Read on &rarr;</a>
+  </article>
+</script>
```

Comment:
Very close! right now your author links dont work, and the author filter isnt populating. minor tweaks and this will work beautifully!